packages/ifpack2/src/Ifpack2_Relaxation_def.hpp

```diff
@@ -726,8 +733,10 @@ void Relaxation<MatrixType>::initialize ()
       if (mtKernelHandle_->get_gs_handle () == nullptr) {
         if (PrecType_ == Details::GS2 || PrecType_ == Details::SGS2)
           mtKernelHandle_->create_gs_handle (KokkosSparse::GS_TWOSTAGE);
-        else if(this->clusterSize_ == 1)
+        else if(this->clusterSize_ == 1) {
           mtKernelHandle_->create_gs_handle ();
+          mtKernelHandle_->get_point_gs_handle()->set_long_row_threshold(longRowThreshold_);
```

Comment:
@brian-kelley Does this mean the long row threshold is incompatible or meaningless for cluster size > 1?  Do you think should there be a check above to see if both are set?  Otherwise I believe the threshold is silently ignored.

Reply:
@jhux2 Yes, it's incompatible with cluster GS. I agree there should be a check.